utils/select-fields.js

```diff
@@ -1,10 +1,10 @@
 const _ = require('lodash');
 const formatParamOutput = require('./format-param-output');
 
-module.exports = (req, context) => {
+module.exports = (req, context, sanitizeFields = true) => {
```

Comment:
Can you please make this parameter an object? So to avoid [boolean traps](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap).

Reply:
Added :)